src/resources/assets/useUserAsset.ts

```diff
@@ -18,3 +22,16 @@ export function useUserAsset(uniqueId: string) {
     }
   );
 }
+
+export const getNetworkNativeAssetUniqueId = (chainId: ChainId) => {
+  const network = getNetworkFromChainId(chainId);
+  const { nativeCurrency } = getNetworkObj(network);
+  const { mainnetAddress, address } = nativeCurrency;
+  const uniqueId = mainnetAddress ? `${mainnetAddress}_${Network.mainnet}` : `${address}_${network}`;
```

Comment:
Unless something changed, this is not how it's supposed to be.   UniqueIds should be created  as  `${mainnetAddress || address}_${chainId}`

Reply:
Yeah we're using chainId format for new swaps stuff. Will be incrementally adopting this new pattern elsewhere in the app later @greg-schrammel.

Reply:
or wait i'm not sure why this file was touched actually.. this is for old user assets logic

Reply:
@greg-schrammel  for swaps work, we've been duplicating the general `resources` files for assets and placing them here: `__swaps__/screens/Swap/resources/assets`; we hadn't copied over the `useUserAsset` function but we should be doing that for now (clean up of this duplication of `resources` will be tackled later)

Reply:
we have a `useUserAsset` that selects a single asset from `useUserAssets`, I created in the same file a `useUserNativeNetworkAsset` to make it easier to grab the network native asset
is there another hook/way of doing that already?
don't see a reason to clone this file